bulk-model-sync-gradle/src/main/kotlin/org/modelix/model/sync/bulk/gradle/config/ModelSyncGradleSettings.kt

```diff
@@ -93,9 +98,6 @@ sealed interface LocalEndpoint : SyncEndpoint {
 
     override fun getValidationErrors(): List<String> {
         val errors = mutableListOf<String>()
-        if (mpsHome == null) {
-            errors.addUndefinedLocalFieldError("mpsHome")
-        }
```

Comment:
We should [document](https://github.com/modelix/modelix.core/blob/main/docs/global/modules/core/pages/reference/component-bulk-model-sync-gradle.adoc?plain=1#L86-L88), what happens if `mpsHome` is null.

Reply:
Done